WordPress/src/main/java/org/wordpress/android/ui/posts/PostPreviewFragment.java

```diff
@@ -86,7 +88,7 @@ void refreshPreview() {
         new Thread() {
             @Override
             public void run() {
-                Post post = WordPress.wpDB.getPostForLocalTablePostId(mLocalPostId);
+                PostModel post = mPostStore.getPostByLocalPostId(mLocalPostId);
```

Comment:
Same as previous comment, we should probably pass the PostModel here: one less id, and we don't need the PostStore here.